packages/recommend-core/src/utils/computeAverageIndexes.ts

```diff
@@ -0,0 +1,21 @@
+import { AverageIndexes } from '../types/AverageIndexes';
+import { IndexTracker } from '../types/IndexTracker';
+
+export const getAverageIndexes = (
+  indexTracker: IndexTracker,
+  nrOfObjs: number
```

Comment:
Maybe it's worth documented the props? When I first read this file I wasn't sure what `nrOfObjs` was.